module/x/gravity/keeper/keeper_valset.go

```diff
@@ -87,14 +87,6 @@ func (k Keeper) SetLatestValsetNonce(ctx sdk.Context, nonce uint64) {
 	store.Set(types.LatestValsetNonce, types.UInt64Bytes(nonce))
 }
 
-// IncrementLatestValsetNonce increments the latest valset nonce in the store and returns it
```

Comment:
This function was unused (excluding tests) so I have removed it

Reply:
where is k.SetLatestValsetNonce called now? the latest valset nonce is a different value in the store than the latest vaset. At least as far as I remember.

Reply:
When we call StoreValset() or StoreValsetUnsafe(), the nonce in the valset is used to update the latest nonce. I noticed that my calls to StoreLatestValsetNonce() were redundant so I removed them:

https://github.com/althea-net/cosmos-gravity-bridge/blob/3bec1385f86ffaaddb737ec2e32f762c5dc8d9b5/module/x/gravity/keeper/keeper_valset.go#L52
https://github.com/althea-net/cosmos-gravity-bridge/blob/3bec1385f86ffaaddb737ec2e32f762c5dc8d9b5/module/x/gravity/keeper/keeper_valset.go#L59